custom/enikshay/integrations/nikshay/repeater_generator.py

```diff
@@ -337,6 +348,102 @@ def handle_exception(self, exception, repeat_record):
                                 "followup_nikshay_registered", "followup_nikshay_error")
 
 
+@RegisterGenerator(NikshayRegisterPrivatePatientRepeater, 'case_xml', 'XML', is_default=True)
```

Comment:
This probably doesn't matter as it isn't going to get shown to the user anywhere, but I'm still hung up on the fact that this payload generator doesn't generate XML, but instead generates a dict to be consumed by the `SOAP` library. @dannyroberts what do you think?